.gitignore

```diff
@@ -1,3 +1,5 @@
 /build
 /out
 CMakeUserPresets.json
+CMakeCache.txt
+CMakeFiles/
```

Comment:
Why these files?

Reply:
Garbage generated be in-source build which `cmake .`

Reply:
I don't think we should add to our `.gitignore` to cover cases where users do unsupported things.

Every line of code we add has an associated maintenance cost and an overhead for readers. I don't think, in this instance, the cost justifies the benefit.

Reply:
see https://github.com/bemanproject/inplace_vector/pull/33#discussion_r1872664908

Reply:
I'm still unconvinced adding these lines is worth the complexity.